### PR TITLE
Raise if call to sixpack fails

### PR DESCRIPTION
**Pull request by drnluz**

### CODE REVIEW

File: lib/sixpack.rb [hunk 1]
```diff
@@ -81,12 +88,9 @@ def participate(experiment_name, alternatives, force=nil, kpi=nil, traffic_fract
...
       end
 
-      res = self.get_response("/participate", params)
-      # On server failure use control
-      if res["status"] == "failed"
-        res["alternative"] = {"name" => alternatives[0]}
-      end
-      res
+      response = self.get_response("/participate", params)
+      raise SixpackRequestFailed.new(response) if response["status"] == "failed"
```
Comment (vsh91) [inline lib/sixpack.rb]: @drnluz What do you think about moving this `raise` to `#get_response`?

Comment (drnluz) [reply lib/sixpack.rb]: makes sense, I'll do it

Comment (Valve) [reply lib/sixpack.rb]: thanks, merge when done @drnluz

**Comment by drnluz** [comment]
@vsh91 I've done some changes based on your suggestion can you take a look again please?

**Review by vsh91 — APPROVED** [review]
@drnluz thanks

**Pull request merged**